frontend/components/ui/footer.tsx

```
@@ -0,0 +1,174 @@
+                            <li>
+                                <Link href='/' className='relative list-wrap'>
+                                    Work at Future Learn
+                                </Link>
+                            </li>
+                        </ul>
+                    </div>
+
+                    <div className='mb-6 lg:ml-8 md:mb-0'>
+                        <h2 className='relative pb-4 mb-5 text-2xl font-semibold text-white'>
+                            Our Company
+                            <span className='absolute bottom-0 left-0 w-10 h-1 rounded-full bg-theme-primary'></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)